javascript/env.local

```diff
@@ -0,0 +1,3 @@
+REACT_APP_AUTH0_DOMAIN=cs156-w21-team-5pm-1.us.auth0.com
+REACT_APP_AUTH0_CLIENT_ID=39mlXCrIrUAl2J5xOF4exh7pP8RzFLqV
+REACT_APP_AUTH0_AUDIENCE=https://cs156-w21-team-5pm-1-courses.herokuapp.com
```

Comment:
It looks like you committed your team's `env.local` file because it was improperly named. This file contains secrets specific to your team and shouldn't be published on the internet.

Please remove this file and reset your client ID in your team's Auth0 tenant.

Reply:
I don't know how to reset client ID, I think we can only rotate secrets but we did not expose the secrets.

Reply:
I'm discussing with @btk5h and @andrewhlu whether we really do need to reset the client_id.   I think we might be able to get away with not doing that.

It's still true that we should NOT commit this to GitHub, and in general, it's a bad practice to let values such as this one leak into GitHub repo history (especially on public repos).  But in this case, since it is a client_id that leaked, and not a client secret or an access token, I think we might not need to reset it.